routes/book.js

```diff
@@ -7,9 +7,13 @@ const {
     deleteBookById
 } = require('../controllers/book.js');
 
+const {
+    checkToken
+} = require('../middlewares/auth.js');
+
 const book = express.Router();
 
-book.post('/', addBook); // record new book
+book.post('/', checkToken, addBook); // record new book
 book.get('/:id', getBookById); // select one book by its id
 book.get('/', getBookByQuery); // select book by title or all
 book.patch('/:id', updateBookById); // update book parametres by id
```

Comment:
also you need to add checkToken to delete book

Reply:
In branch "token" token checks are applied.